.github/workflows/lint.yaml

```diff
@@ -66,5 +66,10 @@ jobs:
           cargo install taplo-cli --version 0.9.3
           git ls-files -z -- '*.toml' ':!:**/testdata/**' ':!:src/generated/**' | \
             xargs -0 taplo fmt
+      - uses: hashicorp/setup-terraform@b9cd54a3c349d3f38e8881555d616ced269862dd # v3
+      - name: Format Terraform files
+        run: |
+          git ls-files -z -- '*.tf' ':!:**/testdata/**' ':!:src/generated/**' | \
```

Comment:
maybe worth a section in the contributor doc

https://github.com/googleapis/google-cloud-rust/blob/0987d7526aff3b1a3d001ee22e3c13dc02ab7e9e/doc/contributor/howto-guide-set-up-development-environment.md?plain=1#L183

Reply:
Done